web-ui/src/api/notifications.js

```diff
@@ -46,3 +47,34 @@ export const sendEmail = async (subject, content, html, recipients, cookie) => {
     }
   });
 };
+
+export const emailPDLAssignment = async (member, cookie) => {
+  if (member.pdlId && member.lastName && member.firstName && member.workEmail) {
+    let res = await getMember(member.pdlId)
+    let pdl =
+        res.payload && res.payload.data && !res.error
+            ? res.payload.data
+            : null;
+    if (pdl && pdl.workEmail) {
```

Comment:
This can be shortened to `if (pdl?.workEmail) {`

Reply:
Added.